HintList.py

```diff
@@ -520,51 +520,51 @@ def tokens_required_by_settings(world):
-    'Graveyard Dampe Race Rewards':                                ("racing #Dampe's ghost# yields...^", None, 'dual'),
-    'Graveyard Royal Family Tomb Contents':                        ("inside the #Royal Family Tomb#, darkness and Redeads guard respectively...^", None, 'dual'),
-    'DMC Child Upper Checks':                                      ("at #the crater# a spider in a crate and a single scrub hold respectively...^", None, 'dual'),
-    'Haunted Wasteland Checks':                                    ("#deep in the wasteland# a chest and a spider hold respectively...^", None, 'dual'),
+    'Deku Theater Rewards':                                        ("the #Skull Mask and Mask of Truth# reward...^", None, 'dual'),
+    'HF Ocarina of Time Retrieval':                                ("the #Ocarina of Time# rewards both...^", None, 'dual'),
+    'HF Valley Grotto':                                            ("in a grotto with a #spider and a cow# you will find...^", None, 'dual'),
+    'Market Bombchu Bowling Rewards':                              ("at the #Bombchu Bowling Alley#, you will be rewarded with...^", None, 'dual'),
+    'ZR Frogs Rewards':                                            ("the #Frogs of Zora River#, will reward you with...^", None, 'dual'),
+    'LH Lake Lab Pool':                                            ("inside the #lakeside lab# a person and a spider hold...^", None, 'dual'),
+    'LH Adult Bean Destination Checks':                            ("#riding the bean in Lake Hylia# leads to...^", None, 'dual'),
+    'GV Pieces of Heart Ledges':                                   ("within the #valley#, the crate and waterfall conceal...^", None, 'dual'),
+    'GF Horseback Archery Rewards':                                ("the #Gerudo Horseback Archery#, rewards...^", None, 'dual'),
+    'Colossus Nighttime GS':                                       ("#at the Desert Colossus#, skulltulas at night hold...^", None, 'dual'),
+    'Graveyard Dampe Race Rewards':                                ("racing #Dampe's ghost# rewards...^", None, 'dual'),
+    'Graveyard Royal Family Tomb Contents':                        ("inside the #Royal Family Tomb#, you will find...^", None, 'dual'),
+    'DMC Child Upper Checks':                                      ("in the #crater, a spider in a crate and a single scrub# guard...^", None, 'dual'),
+    'Haunted Wasteland Checks':                                    ("deep in the #wasteland a chest and a spider# hold...^", None, 'dual'),
 
     'Deku Tree MQ Basement GS':                                    ("in the back of the #basement of the Great Deku Tree# two spiders hold...^", None, 'dual'),
-    'Dodongos Cavern Upper Business Scrubs':                       ("in the #upper parts of Dodongo's Cavern# a pair of scrubs sell...^", None, 'dual'),
+    'Dodongos Cavern Upper Business Scrubs':                       ("in #Dodongo's Cavern a pair of scrubs# sell...^", None, 'dual'),
```

Comment:
This can be confused with the two scrubs in the MQ lobby. This is why the hint previously said "upper parts".

Reply:
Is this an issue though? I thought your preference was to not give away MQ status on a dungeon with hints where possible. Then once you enter the dungeon and know its vanilla it is no longer confusing

Reply:
It is an issue because people who don't know there isn't a hint for the MQ lobby scrubs will not know the MQ/vanilla information has been given away until they step inside.